src/yunohost/backup.py

```diff
@@ -152,6 +152,21 @@ def _clean_tmp_dir(retcode=0):
         'hooks': {},
     }
 
+    # Initialize backup list
+    backup_csv_path = os.path.join(tmp_dir, 'backup.csv')
+    try:
+        backup_csv = open(backup_csv_path, 'a')
+    except:
+        logger.error(m18n.n('backup_csv_creation_failed'))
+
+    def add_into_list(source,dest):
+        try:
+            if dest.endswith("/"):
+                dest=dest+os.path.basename(source)
+            backup_csv.write("{0}\t{1}\n".format(source,dest))
```

Comment:
I understand, but the code as is very fragile (a path with a tab and everything is broken: possibly overwriting by accident or attack arbitrary files). So I'd see two solutions:
- find (or write) a robust function to parse CSV in bash
- reject explicitly any argument with a separator (in our case, a tab) in it